shell/platform/windows/flutter_windows_view_unittests.cc

```diff
@@ -1302,7 +1303,9 @@ TEST(FlutterWindowsViewTest, UpdatesVSyncOnDwmUpdates) {
   FlutterWindowsView view(std::move(window_binding_handler));
 
   InSequence s;
+  EXPECT_CALL(*surface_manager.get(), MakeCurrent).WillOnce(Return(true));
   EXPECT_CALL(*surface_manager.get(), SetVSyncEnabled(true)).Times(1);
+  EXPECT_CALL(*surface_manager.get(), MakeCurrent).WillOnce(Return(true));
```

Comment:
This was making a GL context current, thereby causing other tests to complain that a GL context was unexpectedly current.